experiments/resources/back_up_optimizely.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Required by click http://click.pocoo.org/5/python3/
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+export GIT_AUTHOR_NAME="jenkins"
+export GIT_AUTHOR_EMAIL="cstenson@edx.org"
```

Comment:
Name should be passed in from vars, should not be hardcoded

Reply:
@coryleeio I added the GIT_AUTHOR and email as vars to the edx-internal optimizely.yml here but I'm not sure I'm referencing it correctly here in the script file. https://github.com/edx/edx-internal/pull/710/commits/66606a9a8aa3065397fd0387be9a31da42770243 

Can you take a look and let me know if I've done this right?